docs/DevelopmentPlan/DevelopmentPlan.tex

```diff
@@ -95,42 +96,106 @@ \section{Team Member Roles}
 \end{itemize}
 
 
-
-\section{Work flow Plan}
-
+\section{Workflow Plan}
+        A general development workflow will be: \
 \begin{itemize}
-	\item How will you be using git, including branches, pull request, etc.?
-	\item How will you be managing issues, including template issues, issue
-	classification, etc.?
+        \item \textbf{Opening an issue} \\
+
+        Create an issue using \textit{GitHub}'s issue tracker. Whenever there is a team meeting, lecture, or task, a \textit{GitHub} issue should be created using the corresponding templates. An issue created for recording lecture attendance should follow the issue template created for lecture purposes. An issue created for assigning a task (bug fix, adding a new feature) should follow the issue template created for general purpose. An issue created for documenting the team meetings should follow the issue template created for meeting purposes. An issue should have at least one label (e.g. \texttt{bug}, \texttt{documentation}, \texttt{repo management}, etc), and at least one assignee, and project linked to it. After an issue is created, the assignee is responsible for updating the status of the issue inside the \textit{GitHub} project board. There are four different statuses of issues inside the project board and each one of them should be put under the correct column:
+            \begin{itemize}
```

Comment:
Shall we also mention here about the responsibility of the board coordinator's responsibility to manage and supervise the issue status according to the progress update and sprint planning? But overall is excellent, just an advice

Reply:
Workflow will not cover responsibilities of other roles instead of developers themselves, it is a protocol that everyone should follow, but what you mentioned should be covered and well defined in the team's responsibility, which is part that Jingyao is working on.